src/config/delegation.cpp

```
@@ -0,0 +1,102 @@
+}
+
+/// Load delegation config from file. Simple YAML subset: "  key: value"
+static std::unordered_map<std::string, DelegationTarget> load_config() {
+  std::unordered_map<std::string, DelegationTarget> roles;
+  // Try local override first (gitignored, has hostnames), then shared config
+  std::ifstream f(".config/delegation.local.yml");
+  if (!f.is_open()) {
+    f.open(".config/delegation.yml");
+  }
+  if (!f.is_open()) {
+    const char* home = std::getenv("HOME");
```

Clang (14.0.6)

[warning] variable 'home' is not initialized

(cppcoreguidelines-init-variables)